doc/source/io.rst

```diff
@@ -644,6 +644,15 @@ file, either using the column names, position numbers or a callable:
     pd.read_csv(StringIO(data), usecols=[0, 2, 3])
     pd.read_csv(StringIO(data), usecols=lambda x: x.upper() in ['A', 'C'])
 
+The ``usecols`` argument can also be used to specify which columns not to
+use in the final result:
+
+.. ipython:: python
+   pd.read_csv(StringIO(data), usecols=lambda x: x.upper() not in ['A', 'C'])
```

Comment:
I would simply do `x: x not in ['a', 'c']` here, to focus on the applications of skipping the listed cols (the upper is already in the example above).